docs/pages/docs/core-concepts/remote-caching.mdx

```diff
@@ -42,7 +42,7 @@ Your cache artifacts will now be stored locally _and_ in your Remote Cache.
 To verify, delete your local Turborepo cache with:
 
 ```sh
-rm -rf ./node_modules/.cache/turbo
+npx rimraf ./node_modules/.cache/turbo
```

Comment:
Yes the intent would be to have a command that worked for users that use win based systems too.
Since we need to have node to use turbo and on the examples above the commands run via npx i did the same for this one, but if the download of a package is not desired the `tabs` could be a better solution 👍

Reply:
@mehulkar updated